src/components/Login/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { GoogleLogin } from "react-google-login";
+
+function Login() {
+  const history = useHistory();
+  const googleSuccess = async ({ tokenId }) => {
+    try {
+      const res = await fetch(`${process.env.REACT_APP_API_ADDRESS}/users/login`, {
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${tokenId}`
+        },
+        credentials: "include"
+      });
+      const { data } = await res.json();
+      console.log(data);
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  const googleFailure = () => {
+    history.push("/");
+  };
+
+  return (
+    <GoogleLogin
+      clientId={process.env.REACT_APP_GOOGLE_CLIENT_ID}
+      render={(renderProps) => (
```

Comment:
```suggestion
      render={(props) => (
```
그냥 props도 괜찮은 것 같아요!